app/Http/Controllers/PhotoAlbumController.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Http\Request;
+use App\repositories\PhotoAlbumRepository;
+use App\repositories\RepositorieFactory;
+
+class PhotoAlbumController extends Controller
+{
+    private $_PhotoAlbumRepository;
+    private $_PhotoRepository;
+
+    public function __construct(RepositorieFactory $repositorieFactory){
+        $this->_PhotoAlbumRepository = $repositorieFactory->getRepositorie(RepositorieFactory::$PHOTOALBUMREPOKEY);
+        $this->_PhotoRepository = $repositorieFactory->getRepositorie(RepositorieFactory::$PHOTOREPOKEY);
+        $this->middleware('auth');
+    }
+
+    public function index()
+    {
+        $photoAlbums = $this->_PhotoAlbumRepository->all();
+        foreach($photoAlbums as $photoalbum){
+              $photoalbum->description = str_replace("\r\n","<br>", $photoalbum->description);
+        }
+        $curPageName = trans('front-end/photo.pagetitle');
```

Comment:
Ik zou translations overlaten aan blade in dit geval.

Reply:
Bij alle pagina's gaat dit zo, dat is de variabele die gebruikt wordt om de pagina titel op te bouwen in blade. Voor sommige pagina's is de titel namelijk niet een simpele translatie. (zoals bij de foto album titel)